wandb/internal/stats.py

```diff
@@ -7,6 +7,7 @@
 import threading
 import wandb
 from wandb import util
+from wandb.lib.tpu import is_tpu_available, TPUProfiler
```

Comment:
1)
tpu isnt something used outside the interenal process,  lets put it in internal for now,  we will need an internal/lib at some point to make things cleaner

2)
We are trying to move to module imports only.  So this should be:
```
from wandb.internal import tpu
...
if tpu.is_tpu_available():
```
https://google.github.io/styleguide/pyguide.html#22-imports